server/routerlicious/packages/services-client/src/summaryTreeUploadManager.ts

```diff
@@ -63,6 +64,17 @@ export class SummaryTreeUploadManager implements ISummaryUploadManager {
 			}),
 		);
 
+		if (summaryTree.groupId !== undefined) {
```

Comment:
> @jatgarg Why do we need to do anything to round trip groupId which is just part of the summary tree as a prop? Should not it just show up? That is what I would imagine.

Moving the question here for the thread. The problem I'm trying to solve is that we need to store the groupId in a file/blob. Otherwise we need to store it in the GitManager. The issue with the GitManager is that it's a different interface and it's server based so it would affect the services.

@znewton do you have any feedback here?

Reply:
But do we do any such thing for lets say "unreferenced" prop on the summary? We store the whole summary and groupId is just part of it, no? Why do we need to store it separately?

Reply:
`unreferenced` isn't round tripped. Maybe we should just use whole summary upload for local driver?

Reply:
Also, I see that there is 	unreferenced prop on ISnapshotTree too.

Reply:
What I would recommend is just use whatever is easier as it is for local server and if moving to whole summary is not easy then we should just fit groupid in current summary logic. 

Also, with this you are able to store groupIds for every tree which contains it, right?

Reply:
For posterity: `WholeSummaryUpload` is not supported by the `TestHistorian` currently, we would need to build it out.